Start of dashboard i18n logic & implement on new Auth pages (#372)
astrolicious/studiocms: www/docs/src/content/docs/contributing/getting-started.mdx

---
title: Getting Started
description: Learn how to contribute to StudioCMS
sidebar:
  order: 1
---
import ContributorList from '~/components/ContributorList.astro';

Thank you for investing your time in contributing to our project!

Read our [Code of Conduct](https://github.com/astrolicious/studiocms?tab=coc-ov-file#code-of-conduct-) to keep our community approachable and respectable.

In this guide you will get an overview of the contribution workflow from opening an issue, creating a PR, reviewing, and merging the PR.

## Our Contributors

Our project exists thanks to all the people who contribute. [Join us on GitHub](https://github.com/astrolicious/studiocms) and add your name to the list!

<ContributorList />

## Getting started

> We welcome contributions from the community! Whether it's bug reports, feature requests, or code contributions, we appreciate your help in making this project better. To navigate our codebase with confidence, see [How it works](/how-it-works/) section.

Reviewer: This section's heading repeats the page heading. The paragraph could probably sit above "Our Contributors" I suppose?

Author: done in [d5e2719](https://github.com/astrolicious/studiocms/pull/372/commits/d5e2719e66d9b4f9f34208440d7f115962e5c34c)

## Bug Reports and Feature Requests

> If you encounter any bugs or have ideas for new features, please open an issue on our [GitHub repository](https://github.com/astrolicious/studiocms). When creating a new issue, please provide as much detail as possible, including steps to reproduce the issue (for bugs) and a clear description of the proposed feature.

Reviewer: ```suggestion
If you encounter a bug or want to suggest a new feature, please open an issue on our [GitHub repository](https://github.com/astrolicious/studiocms). When creating a new issue, please provide as much detail as possible, including steps to reproduce the issue (for bugs) and a clear description of the proposed feature.
```
